### PR TITLE
Fix bulk_update

### LINKED ISSUE #507: Bulk update on models containing MoneyField gives error
I am trying to perform bulk updates on the django model that contains MoneyField. This is throwing the below error. object.save() works fine. Anyone encountered this issue? Any possible fixes?
```
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/ghegde/.env/prosper-Kpl-5Ztk/lib/python3.7/site-packages/django/db/models/query.py", line 525, in bulk_update
    self.filter(pk__in=pks).update(**update_kwargs)
  File "/Users/ghegde/.env/prosper-Kpl-5Ztk/lib/python3.7/site-packages/djmoney/models/managers.py", line 205, in wrapper
    args, kwargs = _expand_money_kwargs(model, args, kwargs, exclusions)
  File "/Users/ghegde/.env/prosper-Kpl-5Ztk/lib/python3.7/site-packages/djmoney/models/managers.py", line 147, in _expand_money_kwargs
    value = prepare_expression(value)
  File "/Users/ghegde/.env/prosper-Kpl-5Ztk/lib/python3.7/site-packages/djmoney/utils.py", line 33, in prepare_expression
    if isinstance(expr.rhs, F):
AttributeError: 'Cast' object has no attribute 'rhs'

```

### PR DESCRIPTION
**Pull request by satels**
Close #507 and double issue #533

### CODE REVIEW

**Comment by satels** [comment]
Yes, of course

**Comment by satels** [comment]
Done

**Comment by satels** [comment]
I should add the record to changelog also

**Comment by satels** [comment]
I also changed the version

File: djmoney/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-__version__ = "1.0"
+__version__ = "1.1"
```
Comment (Stranger6667) [inline djmoney/__init__.py]: I'd rather avoid setting version in pull requests - the release date might be different and also we can add more features after this one

File: docs/changes.rst [hunk 2]
```diff
@@ -3,6 +3,14 @@
 Changelog
 =========
 
+`1.1`_ - 2019-12-16
```
Comment (Stranger6667) [inline docs/changes.rst]: Could you, please change these lines into this?

```
`1.1`_ - TBA
------------
```

File: tests/test_models.py [hunk 3]
```diff
@@ -456,6 +456,31 @@ def test_invalid_expressions_access(self, f_obj):
 
 
 class TestExpressions:
+    def test_bulk_update(self):
+        if VERSION[:2] < (2, 2):
```
Comment (Stranger6667) [inline tests/test_models.py]: Usually, we skip tests via `pytest.mark.skipif` decorator - I'd prefer to have it here as well

File: tests/test_models.py [hunk 4]
```diff
@@ -456,6 +456,31 @@ def test_invalid_expressions_access(self, f_obj):
...
+        ModelWithVanillaMoneyField.objects.create(money=Money(2, "USD"), integer=1)
+        first_instance = ModelWithVanillaMoneyField.objects.get(integer=0)
+        second_instance = ModelWithVanillaMoneyField.objects.get(integer=1)
+        assert first_instance.money == Money(1, "USD")
+        assert second_instance.money == Money(2, "USD")
+        first_instance.money = Money(3, "RUB")
+        second_instance.money = Money(4, "UAH")
+        second_instance.second_money = Money(5, "BYN")
+        ModelWithVanillaMoneyField.objects.bulk_update([first_instance, second_instance], ("money", "second_money"))
+        ModelWithVanillaMoneyField.objects.get(integer=0).money == Money(3, "RUB")
```
Comment (Stranger6667) [inline tests/test_models.py]: Missed `assert` statements?

File: tests/test_models.py [hunk 5]
```diff
@@ -456,6 +456,31 @@ def test_invalid_expressions_access(self, f_obj):
...
+        ModelWithVanillaMoneyField.objects.get(integer=1).money == Money(4, "UAH")
+        ModelWithVanillaMoneyField.objects.get(integer=1).second_money == Money(5, "BYN")
+
+    def test_date_lookup(self):
```
Comment (Stranger6667) [inline tests/test_models.py]: This one is identical to the test on line 180 in this file - could you, please, remove it?

File: README.rst [hunk 6]
```diff
@@ -299,7 +299,7 @@ Install the required packages:
 
     cd ./django-money/
 
-    pip install -e .[test] # installation with required packages for testing
+    pip install -e ".[test]" # installation with required packages for testing
```
Comment (benjaoming) [inline README.rst]: Thanks, haha, this has been annoying far too long - have seen this instruction missing the quotes in many other projects.

File: djmoney/__init__.py (hunk 1 above)
Comment (benjaoming) [reply djmoney/__init__.py]: Just ranting here...

I agree that PRs should contain version changes, those we handle separately - also a 1.1 release would be misleading if it only contains a bug fix. I think when doing minor version bumps, the dependency wouldn't traverse very far, and people would feel obliged to read release notes looking for breakage... and of course one doesn't do that when there isn't time.

So ultimately, a minor version bump can be in the interest of non, if it isn't substantial :)

**Comment by satels** [comment]
I will make changes tomorrow.

File: README.rst (hunk 6 above)
Comment (satels) [reply README.rst]: Can I mark this conversation as resolved?

**Comment by satels** [comment]
Ok, this pull request is ready to merge

**Comment by Stranger6667** [comment]
Thanks! Could you, please, rebase? I can't do automatic rebase due to conflicts

**Comment by satels** [comment]
I did rebase for that

File: docs/changes.rst [hunk 7]
```diff
@@ -3,6 +3,14 @@
 Changelog
 =========
 
+`1.1`_ - TBA
+
+Added
+~~~~~
+
+- Support for money QuerySet.bulk_update(). (`satels`_)
```
Comment (benjaoming) [inline docs/changes.rst]:
```suggestion
- Support for ``Money`` type in ``QuerySet.bulk_update()``. (`satels`_)
```

**Comment by cloudcode-hungary** [comment]
Is there an ETA for PyPi release?

**Comment by cloudcode-hungary** [comment]
Guys, is there any eta to release the 1.1 version to pypi? I would really appreciate to have this fix.

**Comment by benjaoming** [comment]
Reviewed the changes and https://github.com/django-money/django-money/compare/1.0...master seems fine. Wondering why it got bumped to 1.1 since the current changes don't have anything backwards incompatible or big big features.

But I guess it's fine, we can be generous with the version numbers now that we're past 1.0 :)

**Comment by benjaoming** [comment]
@Stranger6667 shall we?

**Comment by Stranger6667** [comment]
Let's do it :)

**Comment by benjaoming** [comment]
Cool, pushing some buttons now

**Pull request merged**